src/include/btree_inline.h

```diff
@@ -1756,6 +1757,16 @@ __wt_page_release(WT_SESSION_IMPL *session, WT_REF *ref, uint32_t flags)
     if (F_ISSET(btree, WT_BTREE_IN_MEMORY))
         return (0);
 
+    /*
+     * If it is switched on, the session debug "release evict page configuration" will evict pages
+     * as they are released and become no longer required.
+     */
+
+    if (F_ISSET(session, WT_SESSION_DEBUG_EVICT_RELEASE)) {
```

Comment:
The config option is `release_evict`, it would be better to align the macro as `DEBUG_RELEASE_EVICT`?

Reply:
second that